content/docs/300-on-brand-menu-api/400-reference/100-get-menus.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: GET /v1/menus
+title: Get Menus
```

Comment:
The new groups endpoint causes an overflow due to the long path. Switch to using shorter titles for all endpoints